packages/pyright-internal/src/localization/package.nls.ru.json

```diff
@@ -660,7 +662,6 @@
         "classImplicitlyProtocol": "Класс \"{type}\" неявно класс-протокол, потом что наследован от класса-протокола, но не заполнил все абстрактные символы. Если это сделано намеренно, добавьте `Protocol` или `ABC` к его родительским классам или используйте `metaclass=ABCMeta`.",
         "functionDecoratorTypeAny": "Тип функции неизвестен, потому что декоратор имеет тип Any",
         "lambdaReturnTypeAny": "Лямбда-функция возвращает Any",
-        "noneNotUsableWithAsync": "\"None\" нельзя использовать в блоке \"async with\"",
```

Comment:
This is a duplicate key, not sure how it got into this file

Reply:
so stupid how duplicate keys don't cause a parse error